### ISSUE TITLE
Trailing comma in argument list treated as (undefined) argument

### ISSUE THREAD

**Issue by als-wobe** [kozhushman/pydocstring#4]
Type: Bug

Reproduction: 
Using the extension on
``````python
def test(t: str,):
    ...
``````
results in 
``````python
def test(t: str,):
    """
    Description of test

    Args:
        t (str):
         (undefined):

    """
    ...
``````
Expected result:
``````python
def test(t: str,):
    """
    Description of test

    Args:
        t (str):

    """
    ...
``````

(Probably more usefull for longer argument lists where every argument is
in its own line)

**Comment by kozhushman** [comment]
Thanks for reporting! I will try to update it as soon as possible

**Comment by kozhushman** [comment]
Fixed in release 0.1.0
